resources/assets/uad_lists.json

```
@@ -13368,6 +13392,14 @@
     "labels": [],
     "removal": "Recommended"
   },
+  "com.mediatek.security": {
+    "list": "Misc",
+    "description": "The com.mediatek.security package is part of MediaTek’s security suite, providing features such as malware protection, privacy settings, and secure browsing options. While the package may offer added security features specific to MediaTek devices, there are alternative security solutions available on Google Play that can provide similar or enhanced protection. Removing this package might lead to a loss of MediaTek-specific security features, but it is generally not critical to the basic operation of the device.",
+    "dependencies": [],
+    "neededBy": [],
+    "labels": [],
+    "removal": "Recommended"
```

```suggestion
    "removal": "Expert"
```

If it can be disabled from Settings.


```suggestion
    "removal": "Advanced"
```

If it can't be disabled from Settings.

---

I wouldnt mark Recommended to a security package, security isnt only about virus manager, it may affect things like firewall and updates in the device that helps security overall. I dont know if it can be disabled by the settings, so Im going to suppose it cant. Marking as Expert.